config/settings/production.py

```diff
@@ -134,6 +134,10 @@
         "task": "adserver.tasks.notify_of_completed_flights",
         "schedule": crontab(hour="0", minute="0"),
     },
+    "every-day-archive-offers": {
+        "task": "adserver.tasks.archive_offers",
+        "schedule": crontab(hour="4", minute="0"),
```

Comment:
I worry a little bit about this overlapping with the indexing tasks. It might be nice to implement a locking pattern here at some point, so we don't overload the DB.

Reply:
That's fair. I think the slack messages will help us see if that's happening.